docs/client_actions.md

```diff
@@ -43,6 +43,8 @@ def increment_count(self):
 )
 def increment_count(self):
     self.global_count += 1
+    ...
+    return CustomDataModel(...)
```

Comment:
Haven't try this out but it looks like there should be a return statement here to make things clearer.